wiki/Rules/Code_of_conduct_for_modding_and_mapping/en.md

```diff
@@ -4,58 +4,32 @@ The **code of conduct for modding and mapping** is a set of rules and guidelines
-  - **Intersubjective Issues** - Not prohibited by rulesets, but are agreed upon as being "wrong" by a majority of people. Should be fixed in most cases unless there are opposing intersubjective arguments with equal size, or if the mapper can exhaustively explain why they feel it is acceptable.
-  - **Subjective Issues** - Something that you believe or think on a personal level. Might become intersubjective if many people agree with it.
-- **Unless the concept behind a beatmap is fundamentally flawed from the start**, modding should aim to improve the map **in its current design** - not force your own style upon it. If you truly believe the map has too many significant issues to address individually, try to give a general statement of why this is the case and what direction you think the map should go in to get back on track.
-- When pointing out "issues", inspect each thoroughly and consider exactly what it is about the particular "thing" you dislike. Then, try to offer a potential solution or improvement and consider exactly how that suggestion will improve the map. If necessary, try multiple suggestions to see which work best — the more help you can offer the mapper, the better.
-
-Suggestions themselves should ideally consist of four main parts, though based on the experience of the mapper and/or your personal relationship to them, you may want to be more lenient:
-
-1. **A statement of the issue itself and where it can be found** - including a timestamp is a huge help so that the mapper can find the part in question quickly.
-2. **A brief explanation of why this causes a problem** - this is essential in convincing a mapper to even consider your suggestion. Mappers are much more receptive to criticism and change when it's accompanied by clear, understandable and valid reasoning.
-3. **One or more alternative suggestions, accompanied by explanations of how they address the problem(s)** - much like the above, explanation is key here. Even if you can convince the mapper that their current work can be improved, what makes your suggestion better than theirs, or anyone else's?
-4. **The wording (or even language) used should take the mapper into consideration.** Are they a fluent English speaker? If not, stick to simple and easy-to-understand language. If you both speak the same language, by all means write in that language, but try to append an English summary of your mods and mod replies if possible, to ensure everyone stays on the same page. If necessary, support your suggestions with screenshots, illustrations, or even short video clips to help get your point across.
-
-There are any number of ways to present your mod post, so feel free to experiment, but try to keep the visuals simple and formatted in a way that it is easy for the mapper and others to work with.
+- **The goal of modding is to help a mapper improve their work.** If you're not interested in helping people, steer clear of the discussion.
```

Comment:
the first sentence is good but I feel like the second is oddly hostile toward people who would be in a map discussion that aren't necessarily there to help mod. especially when the map is going for ranked, there's a whole separate conversation about whether the map should be ranked, irrespective of improvements that would typically come through modding

Reply:
Should also have some kind of exception for moderation-related posts like "This BG might not follow the VCC" or "This song might be too political".

Reply:
I believe this the same sentence from the current Code of Conduct, word for word. I do not know however when should people be allowed in a discussion when they're *not* in the mindset of helping people out or holding a healthy discussion. These discussions occur yeah, but if they do people should always keep in mind they're doing it on the basis of helping a mapper's work, not to trash talk it into oblivion without giving any pointers on how to improve it.

Maybe it could be reworded as to not exclusively allow modding, but rather a general guideline meant to encourage healthy discussions, and discourage those who aren't interested in behaving.

> Should also have some kind of exception for moderation-related posts like "This BG might not follow the VCC" or "This song might be too political".

I'm a bit lost, what is this exception supposed to mean? Either way, moderation posts are meant to be done by the GMT and NAT. Users are encouraged to report these issues to them, who will then be discussed and handled by the usergroups respectively. Feels out of scope to include for that reason imo.

Reply:
Regular users can absolutely make moderation-related posts as well as reports - I've done that more than once.

Reply:
Made one small change to potentially tackle the issue. The discussion isn't exclusive for modding that's true. This should better encompass that if a discussion of any kind is to be taken place, it should be expected to be done in good faith.

Reply:
i guess that's fine after the edit. my point was that posting in good faith and trying to help the game overall in some way doesn't always imply modding / helping the mapper improve one map.